### PR TITLE
Correction in Readme

### PR DESCRIPTION
**Pull request by kickb71**
Correction's in the installation file is completed.

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
```
Comment (rathorevaibhav) [inline app/Http/Controllers/DashboardController.php:1]: I don't think this file is needed in this Pull Request. This PR should only include changes related to readme.

File: docs/installation.md [hunk 2]
```diff
@@ -8,21 +8,34 @@ git clone https://github.com/coloredcow-portal/portal
...
+```
+Note: Use the php version (7.3 / 7.4) or else use the below link to download.
+link(https://www.php.net/downloads.php#v7.4.22).
+```
 npm install
 ```
-
-3. npm build
+3. Change's in the php.ini file,
+Go to your php.ini file(C:\xampp\php)
```
Comment (rathorevaibhav) [inline docs/installation.md:18]:
```suggestion
3. Enable the OpenSSL extension. Go to your php.ini file (C:\xampp\php)
```

File: docs/installation.md [hunk 3]
```diff
@@ -8,21 +8,34 @@ git clone https://github.com/coloredcow-portal/portal
...
+Search for openssl,
```
Comment (rathorevaibhav) [inline docs/installation.md:19]:
```suggestion
Search for openssl
```

File: docs/installation.md [hunk 4]
```diff
@@ -8,21 +8,34 @@ git clone https://github.com/coloredcow-portal/portal
...
 composer install
+```
+Note: Use the php version (7.3 / 7.4) or else use the below link to download.
+link(https://www.php.net/downloads.php#v7.4.22).
+```
 npm install
 ```
-
-3. npm build
+3. Change's in the php.ini file,
```
Comment (rathorevaibhav) [inline docs/installation.md:17]: I think this shouldn't be part of installation, but rather pre-requisites file. When setting up XAMPP.

File: resources/views/graph.blade.php [hunk 5]
```diff
@@ -0,0 +1,109 @@
+@extends('layouts.app')
```
Comment (rathorevaibhav) [inline resources/views/graph.blade.php:1]: this file shouldn't be in this Pull Request.

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: resources/views/graph.blade.php (hunk 5 above)
Comment (kickb71) [reply resources/views/graph.blade.php:1]: Okay sir

File: app/Http/Controllers/DashboardController.php (hunk 1 above)
Comment (kickb71) [reply app/Http/Controllers/DashboardController.php:1]: Okay sir

**Comment by kickb71** [comment]
The files have been changed sir

**Review by kuldeep3 — CHANGES_REQUESTED** [review]
The PR says correction in Readme, although I can see other files too that are not required in this PR. Moreover, there is a conflicting file. Please resolve the conflicts and remove the files that are not required in this PR before assigning it for review.

**Pull request closed without merging**